test/functional/kernel/nested-loop/test-kernel-nested-loop.cpp.in

```diff
@@ -29,7 +29,8 @@ using SequentialKernelNestedLoopExecPols = camp::list<
 
     // Depth 3 Exec Pols
     NestedLoopData<DEPTH_3, RAJA::seq_exec,  RAJA::seq_exec, RAJA::seq_exec >
-  >;
+
+>;
```

Comment:
Can we revert the formatting here please, it looks odd

Reply:
They somehow got misaligned in my editor, I'll change it back.